### PR TITLE
fixed issue #196 (changed the word location to place)

### PR DESCRIPTION
**Pull request by naledi**
# Summary

Fixes issue #196

Here's a description of changes:
- changed the name of the a variable
  we replaced the variable location with the variable place.

cc @gdisf/admins
With the help of Lapo Chirici

### CODE REVIEW

File: javascript/exercises/variables.html [hunk 1]
```diff
@@ -15,18 +15,18 @@
 <h3 class="page-header">The Fortune Teller</h3>
 <p><em>Why pay a fortune teller when you can just program your fortune yourself?</em></p>
 <ul>
- <li>Store the following into variables: number of children, partner's name, geographic location, job title.
```
Comment (ejconlon) [inline javascript/exercises/variables.html]: "location" is more natural in the sentence here.  Can you revert this line and keep the changes below?

(maybe `geoLocation` would be an even better variable name!)

Comment (trucnguyen) [inline javascript/exercises/variables.html]: I agree, I'd keep location or geoLocation as a variable name.

Comment (trucnguyen) [inline javascript/exercises/variables.html]: oic, location is probably a reserved keyword in the language 😊 geoLocation is more descriptive than place I think :)

File: javascript/index.html [hunk 2]
```diff
@@ -111,7 +111,8 @@ <h1 class="page-header">Intro to JavaScript: Arrays, Objects, Functions</h1>
         <li><a href="exercises/forloops.html">Exercise: for loops</a>
         <li><a href="exercises/arrays.html">Exercise: arrays</a>
         <li><a href="exercises/wordguesser.html">Homework: The Word Guesser</a>
-        <li><a href="exercises/currencyConverter.html">Homework: The Currency Converter</a>
+        <!--Link leads to nothing-->
```
Comment (brendajin) [inline javascript/index.html]: Please leave the currency converter link in. We have issue #254 to fix the broken link.

**Pull request closed without merging**